fdbserver/Coordination.actor.cpp

```diff
@@ -538,9 +537,15 @@ struct LeaderRegisterCollection {
 	}
 };
 
+StringRef getClusterName(Key key) {
```

Comment:
`ClusterConnectionString::parseKey` already parsed and removed the `:` part. `ccf->getConnectionString().clusterKey()` only contains the ID part. Actually the `description` part, which is the cluster name, is rarely used in the code base, as far as I know.

Reply:
The incoming cluster key format is, e.g. `TestCluster:YXpBMtN0k90xiTZoIdRoeV9F7abDw0fA`, and I wish to remove the prefix before the `:`. Let's take this one offline.